setup.py

```diff
@@ -78,12 +78,12 @@ def read(*names, **kwargs):
     install_requires=[
         "blinker",
         "dill",
-        "numpy",
+        "numpy < 1.24",
```

Comment:
You are working on making it support never numpy versions, right?

Reply:
In my opinion, this should be done before the release of v0.5. However, as the problem was just discovered here, it should be fixed in another PR.